### ISSUE TITLE
JAMS conversion

### ISSUE THREAD

**Issue by carlthome** [erikronstrom/remus#1]
It would be neat to have conversion tools between https://github.com/marl/jams and Remus. Thoughts?

**Comment by erikronstrom** [comment]
Nice! I’ll take a look at it!

Erik

**Comment by erikronstrom** [comment]
Är jams ett format som du använder? Har du i så fall några exempelfiler med faktisk output?

Erik

**Comment by carlthome** [comment]
Yes. It's the expected standard for [mir_eval](https://github.com/craffel/mir_eval), for example, which is very useful because they provide reference implementations of most evaluation metrics for MIREX and so on. There are examples in the [documentation](http://pythonhosted.org/jams/).
